cmd/seednode/main.go

```diff
@@ -147,6 +155,16 @@ func startNode(ctx *cli.Context) error {
 		}
 	}
 
+	restApiPort := ctx.GlobalInt(restApiPortFlag.Name)
```

Comment:
check if set

Reply:
no need for this. it was an int flag (after review it is a string flag) so we cannot tell that the flag was set or the default value was used.